packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardLarge.tsx

```diff
@@ -73,22 +79,26 @@ export const ReleaseCardLarge: React.FunctionComponent<
             <CalendarTodayTwoTone sx={{ color: 'grey.700' }} />
             <Typography variant="headline1">
               {releaseCardConfig.prependRelease && 'Release '}
-              {releaseVersion}
+              {releaseName}
             </Typography>
           </Stack>
           <Stack
             direction={{ xs: 'column', sm: 'row' }}
             alignItems={{ xs: 'flex-start', sm: 'center' }}
             gap="20px"
           >
-            {buttonToExplorePageConfig && pathAndQuery && (
-              <Button variant="contained" color="primary" href={pathAndQuery}>
-                {buttonToExplorePageConfig.label}
+            {primaryBtnConfig && primaryBtnPathAndQuery && (
+              <Button
+                variant="contained"
+                color="primary"
+                href={primaryBtnPathAndQuery}
+              >
```

Comment:
Should use `react-router-dom`'s `Link` to prevent a full page reload since these are always within the portal site

```
import { Link as ReactRouterLink } from 'react-router-dom'
```

```suggestion
              <Button
              	component={ReactRouterLink}
                variant="contained"
                color="primary"
                to={primaryBtnPathAndQuery}
              >
```